build/yaml/js-build-steps.yml

```diff
@@ -0,0 +1,85 @@
+#variables:
+#  PackageVersion: Define this in the calling pipeline.
+
+steps:
+- task: colinsalmcorner.colinsalmcorner-buildtasks.tag-build-task.tagBuildOrRelease@0
+  displayName: 'Tag Build with version number. (Skip if a fork PR to avoid access denied error.)'
+  inputs:
+    tags: |
+     v: $(PackageVersion)
+     preview v: $(PreviewPackageVersion)
+  continueOnError: true
+  condition: and(succeeded(), eq(variables['System.PullRequest.IsFork'], 'false'))
+
+- task: NodeTool@0
+  displayName: 'Use Node $(NodeVersion)'
+  inputs:
+    versionSpec: $(NodeVersion)
+
+- task: Npm@1
+  displayName: 'npm install lerna ...'
+  inputs:
+    command: custom
+    verbose: false
+    customCommand: 'install --global lerna@3.22.1 nyc mocha eslint-plugin-jsdoc'
+
+- task: Npm@1
+  displayName: '/tools npm install'
+  inputs:
+    workingDir: tools
+    verbose: false
+
+- bash: |
+   set -o xtrace
+   sed -i '' 's/${Version}/$(PackageVersion)/g' package.json 
+   sed -i '' 's/${Version}/$(PreviewPackageVersion)/g' libraries/adaptive-expressions/package.json
```

Comment:
in 4.7, this package was called `botframework-expressions`. I will fix this, thanks for pointing it out @EricDahlvang!